Assets/Scripts/Input/Player.cs

```diff
@@ -304,7 +306,9 @@ private IEnumerator GoalAnimation()
             else
             {
                 Run(0);
-                JumpAction();
+                transform.GetChild(0).transform.eulerAngles = new Vector3(0, lookAtCrowd, 0);
```

Comment:
Use a [SerializeField] for this instead of GetChild - just in case we need to add a child